src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -610,8 +630,8 @@ def load_command_table(self, _):
                             server_connection_policies_operations,
                             client_factory=get_sql_server_connection_policies_operations) as c:
 
-        c.show_command('show', 'get')
-        c.generic_update_command('update')
+        c.custom_show_command('show', 'conn_policy_show')
+        c.custom_command('update', 'conn_policy_update')
```

Comment:
Minor issue: function naming is not consistent in SQL, some are named as `noun_verb`, the others are named as `verb_noun`. Usually we name it as `verb_noun`, for example `update_conn_policy`.

Reply:
addressed